SW.Serverless.Contract/Protos/adapter.proto

```
@@ -0,0 +1,137 @@
+syntax = "proto3";
+
+option csharp_namespace = "SW.Serverless.Contract";
+package sw.serverless.v1;
```

Buf (1.72.0)

[error] Files with package "sw.serverless.v1" must be within a directory "sw/serverless/v1" relative to root but were in directory "SW.Serverless.Contract/Protos".

(PACKAGE_DIRECTORY_MATCH)